examples/3d/split_screen.rs

```diff
@@ -0,0 +1,108 @@
+//! Renders two cameras to the same window to accomplish "split screen".
```

Comment:
I just want to note that this PR and specifically this example is something I wanted to be able to add debug meshes for view frusta and such so the debug mesh for one frusta is visible from another view. I think in general this will help me debugging stuff like the cascaded shadow maps. Maybe it was just meant to be that this PR would come first. ;)